impedancefitter/readin.py

```diff
@@ -23,6 +23,65 @@
 
 logger = logging.getLogger(__name__)
 
+def _readin_Data_from_formated_dataframe(formated_df, minimumFrequency=None, maximumFrequency=None):
+    # sort frequencies to be on the safe side
+    tmp = formated_df.values
```

Comment:
Why do you extract the values? 
It would be much easier to just use the columns, right?
If you add two keywords that identify the right columns (frequency, real and imaginary part) that should be possible, right?